server/stt/stt_json_planning_formatter.py

```diff
@@ -44,8 +44,69 @@ def update_stt_urgency(self, item):
                 }
             )
 
+    def exclude_internal_planning_fields(self, item):
+        if "internal_note" in item:
+            del item["internal_note"]
+        if "subject" in item:
+            item["subject"] = [
+                subj for subj in item["subject"] if subj.get("scheme") != "sttcheckedby"
+            ]
+
+    def exclude_internal_coverage_fields(self, item):
+        exclude_fields = [
+            "headline",
+            "location",
+            "sttpicturewhatabout",
+            "sttpicturewhatisphotographed",
+            "internal_note",
+            "sttdoesphotographerknow",
+            "sttpictureregistrationok",
+            "sttregistrationinfo",
+            "sttpicturecategory",
+            "sttpictureheadlinefi",
+            "sttpicturecaptionfi",
+            "sttpictureinstructionsfi",
+            "sttpicturekeywordsfi",
+            "sttpictureheadlineen",
+            "sttpicturecaptionen",
+            "sttpictureinstructionsen",
+            "sttpicturekeywordsen",
+            "sttpictureinvoiced",
+        ]
+
+        if "coverages" in item:
+            for coverage in item["coverages"]:
+                if "planning" in coverage:
+                    planning = coverage["planning"]
+                    # First remove direct attributes in the planning obj
+                    for field in exclude_fields:
+                        if field in planning:
+                            del planning[field]
+                    # Then filter out internal fields from the fields array
+                    if "fields" in planning:
+                        planning["fields"] = [
+                            f
+                            for f in planning["fields"]
+                            if f.get("field") not in exclude_fields
+                        ]
+
+    def map_agenda_output(self, item):
+        anpa_category = item.get("anpa_category", [])
+        agendas = []
+        for category in anpa_category:
+            agendas.append({"_id": category["name"], "name": category["name"]})
```

Comment:
I know the ticket suggests to use `name` for the `_id`. In CVs, `name` can change but we don't support changing the `qcode`. Therefor it might be good to use `qcode` here for `_id` instead (I've placed a comment in the ticket)